solidity/contracts/Mailbox.sol

```diff
@@ -135,6 +145,7 @@ contract Mailbox is
     function process(bytes calldata _metadata, bytes calldata _message)
         external
         override
+        notPaused
```

Comment:
I believe we can just do `nonReentrant` right? And maybe call it `nonReentrantAndNotPaused` or something? Because this will fail if it's paused:
```
require(_status == _NOT_ENTERED, "ReentrancyGuard: reentrant call");
```

Otherwise we're gonna do an SLOAD in notPaused and then another in nonReentrant (I guess this will be a warm one, so it's far cheaper)

Reply:
yeah was thinking of keeping them explicit but you are right the `notPaused` is unnecessary